Assets/Scripts/Source/Game/Combat/Weapon.cs

```diff
@@ -13,9 +13,23 @@ public virtual int Defense
         private set;
     }
 
-    public Weapon(string name, int damage, int defense) : base(name)
+    public virtual int BonusAttack
+    {
+        get;
+        private set;
+    }
+
+    public virtual int BonusDefense
+    {
+        get;
+        private set;
+    }
+
+    public Weapon(string name, int damage, int defense, int bonusAttack, int bonusDefense) : base(name)
```

Comment:
Hmmm: I wonder if it's cleaner to just pass an `Attack` and `Defense` instance into `Weapon`? Otherwise, we just have to add a lot of duplicate params, e.g. for bonuses